go/client/logclient.go

```diff
@@ -308,3 +309,11 @@ func (c *LogClient) GetSTH() (sth *ct.SignedTreeHead, err error) {
 	sth.TreeHeadSignature = *ds
 	return
 }
+
+// GetSTHConsistency retrieves the consistency proof between two snapshots.
+func (c *LogClient) GetSTHConsistency(first, second uint64) ([][]byte, error) {
```

Comment:
The others aren't protocol buffers, aren't they? They're just structures declared in a common package? I think this is okay, since proofs aren't liable to be kept or passed around much...